mne/epochs.py

```diff
@@ -1385,8 +1393,17 @@ def _get_data(self, out=True, picks=None, item=None, *, units=None,
         else:
             # we start out with an empty array, allocate only if necessary
             data = np.empty((0, len(self.info['ch_names']), len(self.times)))
-            logger.info('Loading data for %s events and %s original time '
-                        'points ...' % (n_events, len(self._raw_times)))
+            msg = (f'for {n_events} events and {len(self._raw_times)} '
+                   'original time points')
+            if self._decim > 1:
+                msg += ' (prior to decimation)'
+            _msg_preload = False
+            if hasattr(self._raw, "preload"):
+                _msg_preload = self._raw.preload
+            if _msg_preload:
+                logger.info(f'Using data from preloaded Raw {msg} ...')
+            else:
+                logger.info(f'Loading data {msg} ...')
```

Comment:
for non-preloaded non-decimated:

> Loading data for X events and Y original time points ...

with decim > 1:

> Loading data for X events and Y original time points (prior to decimation) ...

preloaded: 

> Using data from preloaded Raw for X events and Y original time points ...

with decim > 1:

> Using data from preloaded Raw for X events and Y original time points (prior to decimation) ...

---

if `self._raw` is `None` or a list of `_RawContainer`, then I assume `preload=False` ... else I check it explicitly